### DISCUSSION TITLE
Confusion in explorer

### DISCUSSION THREAD

**Discussion by therealharpaljadeja** [celo-org/social-connect#251] [Help (Q&A)]
This can be a bit confusing even though I interacted with `FederatedAttestationsProxy` the explorer says I interacted with `FederatedAttestations`

I understand under the hood I interacted with it but still :sweat_smile:

[Transaction Link](https://explorer.celo.org/alfajores/tx/0x1ccf69464f61c6a2cf5680bf143bd0ebeb00bd6139b181c2309109cce1115371)

![image](https://github.com/celo-org/SocialConnect/assets/38040789/4c46ad56-821f-4b49-a9c7-0989e45fdac9)

![image](https://github.com/celo-org/SocialConnect/assets/38040789/f96269d6-71f4-4b9a-9a4b-4f3bcaacc54c)

**Comment by arthurgousset** [comment]
Thanks for sharing!
